### PR TITLE
Update local launch script to include checks

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: 9986
Story: 9972

### CODE REVIEW

File: launch-or-update-local.ps1 [hunk 1]
```diff
@@ -32,7 +32,37 @@ param(
...
+    exit 1
+}
+
+# Add namespace if missing
+$namespacePresent = kubectl get namespace $namespace
+if (!($namespacePresent))
+{
+    kubectl create namespace $namespace
+    write-host "`nNamespace has been created`n"
+}
```
Comment (PhysicsJon) [inline launch-or-update-local.ps1:55]: Would we want to do this in the same way that the selenium grid does it? (i.e. `kubectl apply -f local-namespace.yml`)

File: launch-or-update-local.ps1 [hunk 2]
```diff
@@ -32,7 +32,37 @@ param(
...
+
+# Check for Azure repo creds
+$regCredentials=kubectl get secret regcredlocal -n $namespace
+if (!($regCredentials))
+{
+    write-host "Missing Credentials for ACR - please follow instructions in docs\k8s-private-registry.md"
+    write-host "Exiting...."
+    start-sleep 20
```
Comment (PhysicsJon) [inline launch-or-update-local.ps1:63]: What benefit does this sleep bring here?

**Review by PhysicsJon — CHANGES_REQUESTED** [review]

File: launch-or-update-local.ps1 (hunk 1 above)
Comment (gavinwakeley1) [reply launch-or-update-local.ps1:55]: Not really much difference I think, but no problem - I'll amend the script to run that

File: launch-or-update-local.ps1 (hunk 2 above)
Comment (gavinwakeley1) [reply launch-or-update-local.ps1:63]: It lets you see the error message before PowerShell exits and closes - if launched directly

Comment (PhysicsJon) [reply launch-or-update-local.ps1:63]: ahh, seems sensible then

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**